go.mod

```diff
@@ -1,27 +1,28 @@
 module github.com/scaleway/terraform-provider-scaleway/v2
 
 require (
-	github.com/aws/aws-sdk-go v1.44.212
+	github.com/aws/aws-sdk-go v1.44.193
 	github.com/dnaeon/go-vcr v1.2.0
-	github.com/docker/docker v23.0.1+incompatible
+	github.com/docker/docker v20.10.23+incompatible
 	github.com/dustin/go-humanize v1.0.1
 	github.com/google/go-cmp v0.5.9
 	github.com/hashicorp/aws-sdk-go-base v1.1.0
 	github.com/hashicorp/awspolicyequivalence v1.6.0
 	github.com/hashicorp/go-cty v1.4.1-0.20200414143053-d3edf31b6320
 	github.com/hashicorp/go-multierror v1.1.1
 	github.com/hashicorp/go-retryablehttp v0.7.2
-	github.com/hashicorp/terraform-plugin-log v0.8.0
-	github.com/hashicorp/terraform-plugin-sdk/v2 v2.25.0
+	github.com/hashicorp/terraform-plugin-log v0.7.0
+	github.com/hashicorp/terraform-plugin-sdk/v2 v2.24.1
 	github.com/robfig/cron/v3 v3.0.1
-	github.com/scaleway/scaleway-sdk-go v1.0.0-beta.14.0.20230314170003-6858369da2b1
-	github.com/stretchr/testify v1.8.2
+	github.com/scaleway/scaleway-sdk-go v1.0.0-beta.15.0.20230324173156-e992c027fa5f
+	github.com/stretchr/testify v1.8.1
 	golang.org/x/exp v0.0.0-20230118134722-a68e582fa157
```

Comment:
There is many downgrade, there may be issues with your branch

Reply:
I just rebased it from master though :( how can I fix this ?